modules/installation-user-infra-machines-advanced-enabling-serial-console.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+:_content-type: PROCEDURE
+[id="installation-user-infra-machines-advanced-enabling-serial-console_{context}"]
+= Enabling the serial console for PXE and ISO installations
+
+By default, the {op-system-first} serial console is disabled and all output is written to the graphical console. The following procedure for an ISO installation enables the serial console and reconfigures the bootloader so that output is sent to both the serial console and the graphical console.
+
+.Procedure
+
+. Boot the ISO installer.
+
+. Run the `coreos-installer` command to install the system, adding the `--console` option once to specify the graphical console, and a second time to specify the serial console:
++
+[source,terminal]
+----
+$ coreos-installer install \
+  --console=tty0 \ <1>
+  --console=ttyS0,<options> \ <2>
```

Comment:
This should fix the color rendering in the block output:
```suggestion
  --console=tty0 \//<1>
  --console=ttyS0,<options> \//<2>
```

Reply:
thank you, I didn't know how to do this before!